README.md

```diff
@@ -3,10 +3,30 @@
 [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg)](https://www.eclipse.org/legal/epl-2.0/)
 
 ## Installing Codewind for Eclipse
-Complete the following steps to install Codewind for Eclipse:
+Prerequisites
+- Download and install the latest [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/) or use an existing installation. The earliest supported version of the Eclipse IDE for Codewind for Eclipse is 4.11.0 (2019-03).
+- Install Docker.
+- If you use Linux, you also need to install Docker Compose.
 
-1. Download and install the latest [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/) or use an existing installation. The earliest supported version of the Eclipse IDE for Codewind for Eclipse is 4.11.0 (2019-03).
-2. Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+Complete the installation:
+1. Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+2. Open the **Codewind Explorer** view.
+3. Double click on the **Codewind** entry in the view to finish installing Codewind. The download is approximately 1GB.
```

Comment:
- "Double click" needs a hyphen: "Double-click"
- We need a space in between "1GB" so that it's "1 GB".